packages/components/virtual-list/src/builders/build-grid.ts

```diff
@@ -182,7 +182,7 @@ const createGrid = ({
         getEstimatedTotalHeight(props, unref(cache))
       )
       const estimatedTotalWidth = computed(() =>
-        getEstimatedTotalWidth(props, unref(cache))
+        Number.parseInt(getEstimatedTotalWidth(props, unref(cache)), 10)
```

Comment:
Direct rounding should result in loss of precision and inaccurate calculations, which seems to be more likely to result in a larger error than before the modification? 🤔

Reply:
It is indeed possible. 
However, since the `width` is rounded on line 108 and then divided by the `estimatedTotalWidth` to calculate the `ratio` (on line 563), if the `estimatedTotalWidth` is not rounded, it seems that there will always be problems with the calculation result. 
Are there any other more reliable modification schemes? For example: 

- don't round the `width`; 
- round the calculation result of the `ratio`. 

Does the teacher have any modification suggestions?